src/main/kotlin/uk/gov/dluhc/printapi/dto/PrintRequestSummaryDto.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.dluhc.printapi.dto
+
+import uk.gov.dluhc.printapi.database.entity.Status
+import java.time.Instant
+
+data class PrintRequestSummaryDto(
+    private val status: Status,
```

Comment:
@Neil-Massey , I think it will be good to have a separate `Status` enum for the dto to decouple it from the entity enum.

Reply:
Good spot - I'm not convinced with need so many representations of the same enum and mappings, but that's the design 👍

Reply:
Adds so much complexity for what benefit?